### ISSUE TITLE
[Translations] Move translations to Crowdin

### ISSUE THREAD

**Issue by ImRodry** [sp614x/optifine#5336]
## Description of the new feature
Please provide as much detail as possible

At the moment, translations are handled in lang files on this repository, which is not practical for most users. I believe it would be a better idea to move them to Crowdin as this website offers seamless integrations and much more practical tools for translation. If you want to make this change and need help setting up the project, I have a lot of experience with that so I’ll be glad to help. Let me know what you think!

**Comment by ImRodry** [comment]
This was also mentioned in #1403 almost 3 years ago and didn’t get much attention so I’m bringing it up again

**Comment by comradekingu** [comment]
Why recommend Crowdin over Weblate?
Happy to help if the latter is picked.

**Comment by ImRodry** [comment]
> Why recommend Crowdin over Weblate?

Personally never heard of that but Crowdin is the most popular crowdsourcing platform and it offers a lot of customizability with amazing support, so I don't see why not. Many popular projects use it like mc itself, Hypixel and Mineplex

**Comment by comradekingu** [comment]
@ImRodry It is closed source software with egregious terms and conditions, and it is only offered as-a-service. It also holds up tranlsations in a voting system by default, which doesn't produce consistency anywhere I've seen it employed.

https://hosted.weblate.org/projects/minetest/#components is the closest counterpart I suppose.

**Comment by ImRodry** [comment]
> @ImRodry It is closed source software with egregious terms and conditions, only offered as-a-service at that. It also holds up translations in a voting system by default, which doesn't produce consistency anywhere I've seen it employed.
> 
> https://hosted.weblate.org/projects/minetest/#components is the closest counterpart I suppose.

That's not true, Crowdin uses proofreaders to approve strings and that website you linked looks horrible ngl

**Comment by comradekingu** [comment]
What part of it isn't true?

You can use proofreaders in any system, all of which have support for it as a role.

How do you get an overview of many strings and their translations into many languages on Crowdin? How do you ensure everything is in a consistent state without relying on making two accounts to fake-vote bad submissions out of just sitting there with precedence vs. your one vote? How do you get anything done quickly without paging back and forth in Crowdin?

Give the practicality of all available system a shake beyond aesthetics.

**Comment by ImRodry** [comment]
> What part of it isn't true?
> 
> You can use proofreaders in any system, all of which have support for it as a role.
> 
> How do you get an overview of many strings and their translations into many languages on Crowdin? How do you ensure everything is in a consistent state without relying on making two accounts to fake-vote bad submissions out of just sitting there with precedence vs. your one vote? How do you get anything done quickly without paging back and forth in Crowdin?
> 
> Give the practicality of all available system a shake beyond aesthetics.

the votes thing you mentioned isn't true. Crowdin votes are merely indicative for proofreaders and only do something if the project's settings are set to accept non approved translations.
You can switch easily between strings and view all of them individually and all of their translations in all languages at any time. You can replace any word (case sensitive) across the entire project and pick out strings you don't want to replace. as a proofreader/manager you can delete misleading votes. Didn't really understand the last point.

Either way like I said Crowdin is more popular therefore more people are used to it and not some sketchy website

**Comment by comradekingu** [comment]
You don't have proofreaders by default. Last time used Crowdin, translations in other languages isn't the default for users. That works great in the most popular crowd-sourced system, which is Wikipedia. By your own logic that would make translatewiki the best. We digress.

Overall projects on Crowdin have fewer contributors, meaning those aforementioned projects don't really bleed into the mix. What a billion dollar company can do, backed by an even larger entity, isn't really conducive to a like-for-like comparison. You can copy exactly none of that work over, because it is all proprietary btw. Nor is having a lot of translators by number a good idea if there is a lack of overview, but that concerns actual matters of quality again.

What I don't want on a platform beyond being spied on is having to switch views or flip through pages to see everything. I've used Crowdin for years btw, but not lately. Being able to, or having to do various things you can do anywhere as an admin isn't really here nor there when it comes to overview.

As far as 
> and not some sketchy website

goes, Weblate _isn't_ just a website like Crowdin. You don't address this point at all. If you want to say the initial impression of sketchiness is more important than say, the respective terms and services, I suppose you really know what you are talking about as a, designer?

The last point above concerns actually trying the different systems before you make up your mind. That way we are both arguing from the same place.
